api/users/models.py

```diff
@@ -121,6 +121,13 @@ def __str__(self):
     def subscribe_to_mailing_list(self):
         mailer_lite.subscribe(self)
 
+    @hook(BEFORE_DELETE)
+    def delete_organisation_with_a_last_user(self):
```

Comment:
Extreme nitpick: but what do you think about renaming the function to `..with_the_last...` instead of `..with_a_last.`?

Reply:
Thanks @gagantrivedi! what about `delete_single_user_organisations` or `delete_only_user_organisations`?

Reply:
Or it can be also `delete_last_user_organisations` to be aligned with the frontend.